plugin/src/main/kotlin/com/github/triplet/gradle/play/PublishApkTask.kt

```diff
@@ -16,8 +16,9 @@ open class PublishApkTask : PlayPublishPackageBase() {
 
     @TaskAction
     fun publishApks() = write { editId: String ->
-        //TODO: If we take in a folder here as an option, we can fix #233, #227
-        val publishedApks = publishApks(editId)
+        val files = extension.overrideBuildOutput?.listFiles({ _, name -> name.endsWith(".apk") })?.toList()
```

Comment:
The filter should be extracted into an object in a new `Packaging.kt` file:
https://github.com/Triple-T/gradle-play-publisher/blob/591a1138a5e602bfc3d5377776c35d3c33d79e98/plugin/src/main/kotlin/com/github/triplet/gradle/play/internal/Listings.kt#L13-L15